config-controller/internal/controller/policy_controller.go

```diff
@@ -170,7 +176,7 @@ func (r *SecurityPolicyReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 	}
 
 	if err := r.K8sClient.Status().Update(ctx, policyCR); err != nil {
-		rlog.Error(err, "error updating status for securitypolicy '%s'", policyCR.GetName())
+		log.Debugf("error updating status for securitypolicy %q", policyCR.GetName())
```

Comment:
this logging level seems inconsistent with the one above when a status update fails (debug here vs error above).

Reply:
The status update log is a red herring since it happens frequently in the controller per our slack discussion. Printing an error just causes extraneous suspicious but harmless entries in the log, hence debug.

Reply:
That makes sense to me.  Can you update the other status update log to also be debug?